build.fsx

```diff
@@ -102,22 +107,32 @@ let run exe arg dir =
     let result = Process.execWithResult (psi exe arg dir) TimeSpan.MaxValue
     if not result.OK then (failwithf "`%s %s` failed: %A" exe arg result.Errors)
 
+type Communication =
+    | Remoting
+    | Bridge
+    | NoCommunication
```

Comment:
let's use `option` type and `None` for default

Reply:
Huh, another example of how taking less time thinking creates overly complex things. That will even help on the other steps, good catch